primedev/shared/playlist.cpp

```diff
@@ -121,5 +122,7 @@ ON_DLL_LOAD_RELIESON("engine.dll", PlaylistHooks, (ConCommand, ConVar), (CModule
 	module.Offset(0x18ED8D).Patch("C3");
 
 	// patch to allow setplaylistvaroverride to be called before map init on dedicated and private match launched through the game
-	module.Offset(0x18ED17).NOP(6);
+	// only run on non-vanilla as this patch makes users unable to change private match settings on vanilla
+	if (!g_pVanillaCompatibility->GetVanillaCompatibility())
```

Comment:
would rather this byte patch was replaced with a function hook, so that in the case where vanilla compatibility becomes toggleable at runtime this will behave properly.